MobileCenter/MobileCenter/Internals/Sender/MSHttpSender.h

```diff
@@ -23,7 +22,10 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  Pending http calls.
  */
-@property(atomic, strong) NSMutableDictionary<NSString *, id <MSSenderCall>> *pendingCalls;
+@property(atomic, strong) NSMutableDictionary<NSString *, MSSenderCall *> *pendingCalls;
+
+// TODO (jaelim): Add doc here.
+- (void)sendAsync:(NSObject *)container callId:(NSString *)callId completionHandler:(MSSendAsyncCompletionHandler)handler;
```

Comment:
I would try to use a generic type on the class itself and use it here instead of `NSObject`. Then you may want to restrict the type within the `MSIngestionSender`.